src/main/java/com/googlecode/lanterna/TerminalPosition.java

```diff
@@ -159,6 +159,40 @@ public TerminalPosition with(TerminalPosition position) {
         return position;
     }
 
+    public TerminalPosition plus(TerminalPosition position) {
+        return withRelative(position);
+    }
+    
+    public TerminalPosition minus(TerminalPosition position) {
+        return withRelative(-position.getColumn(), -position.getRow());
+    }
+    
+    public TerminalPosition multiply(TerminalPosition position) {
+        return new TerminalPosition(column * position.column, row * position.row);
+    }
+    
+    public TerminalPosition divide(TerminalPosition denominator) {
```

Comment:
I don't see you calling this anywhere? Did you have any particular use case in mind?

Reply:
I am thinking this was for set completeness.